src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx

```diff
@@ -57,6 +65,19 @@ function OrderSubmitStatusAlert({
             } else {
                 return getOrderSubmissionFailedDisplay([isSubmitFailed.message]);
             }
+        } else if (isRetrySubmitIsPending) {
```

Comment:
This condition is never reached because the the condition on line 44 is always true in case of retry because the old data exists already. Because of this, the "in-progress" green alert is never displayed when retry is clicked. 

please consider resetting the query data as part of the retry click.